lib/active_record/virtual_attributes/virtual_fields.rb

```diff
@@ -159,7 +159,8 @@ def call
           records_by_assoc.each do |klass_associations, klass_records|
             next if klass_associations.blank?
 
-            Array[klass_associations].each do |klass_association| # rubocop:disable Style/RedundantArrayConstructor
+            klass_associations = Array[klass_associations] unless klass_associations.kind_of?(Array)
+            klass_associations.each do |klass_association| # rubocop:disable Style/RedundantArrayConstructor
```

Comment:
@kbrock Why is this using `Array[]` as opposed to `Array()`  (Similar question to the rubocop Array redundant constructor).  I think this does the same thing.

```suggestion
            Array(klass_associations).each do |klass_association|
```

Reply:
```ruby
Array({:a => :b}) = [[:a, :b]]
Array[{:a => :b}) = [{:a => :b}]
```

Probably want `Array.wrap`, thought that feels like it has gone out of favor. (validate this statement please)

Reply:
I'm fine with Array.wrap - Ruby is not very consistent, and Array.wrap solves that inconsistency